xcode.mk

```diff
@@ -7,10 +7,22 @@ CMAKE_BOOST_OPTIONS = -DBoost_NO_BOOST_CMAKE=TRUE \
 	-DBOOST_ROOT="$(BOOST_ROOT)"
 endif
 
+export OSX_SDK_PATH = $(shell xcrun --sdk macosx --show-sdk-path)
+
+# https://cmake.org/cmake/help/latest/envvar/MACOSX_DEPLOYMENT_TARGET.html
```

Comment:
The environment variable is only documented since cmake 3.10. That's old enough. So
I'm considering bumping the minimum required version of cmake in librime. Let me handle it.